queue_services/entity-emailer/src/entity_emailer/email_templates/BC-BN.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    {% if title %}
+    <title>{{ title }}</title>
+    {% else %}
+    <title>{{business.legalName}} - Business Number Information</title>
+    {% endif %}
+    [[style.html]]
+  </head>
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+          [[header.html]]
+
+          <div class="container">
+            <p>
+              Canada Revenue Agency has provided a Business number for the company {{ business.legalName }},
```

Comment:
I just chatted with Sienna on this. Please capitalize Business Number and Incorporation Number.